.github/workflows/ci.yaml

```diff
@@ -19,6 +19,7 @@ on:
     branches:
       - "pull-request/[0-9]+"
       - main
+      - ARM_Runner
```

Comment:
This should not be needed. Our GitHub actions run on PRs, so the changes you are making we can test via this PR.

Reply:
@JunAr7112 let's remove this.